fe/src/components/Users/Users.scss

```diff
@@ -1,3 +1,11 @@
-.users {
+$fa-font-path: "../../../node_modules/font-awesome/fonts";
```

Comment:
@ktrzos 
Zadeklarowana stała ale nie użyta, czy jest potrzebna ?

Reply:
Jest użyta :). Wejdź sobie w plik:
```
node_modules/font-awesome/scss/font-awesome/variables
```
zmienna `$fa-font-path` w naszym komponencie nadpisuję tamtą wartość na poprawną. Z resztą, usuń tą linię, to dowiesz się, że się coś zepsuje :).